deps/verifier/Cargo.toml

```diff
@@ -37,13 +50,15 @@ kbs-types.workspace = true
 log.workspace = true
 openssl = { version = "0.10.55", optional = true }
 pv = { version = "0.10.0", package = "s390_pv", optional = true }
-scroll = { version = "0.11.0", default-features = false, features = ["derive"], optional = true }
+scroll = { version = "0.11.0", default-features = false, features = [
+    "derive",
+], optional = true }
 serde.workspace = true
 serde_json.workspace = true
 serde_with = { workspace = true, optional = true }
-sev = { version = "3.1.1", features = ["openssl", "snp"], optional = true }
+sev = { version = "4.0.0", features = ["openssl", "snp"], optional = true }
 sha2.workspace = true 
-tokio = { workspace = true, optional = true, default-features = false }
+tokio = { workspace = true, optional = true }
```

Comment:
why do we want to turn on the tokio default-features?

Reply:
Again, I am not sure. in this case, I am just removing the ignored `default-features = false` flag from the Cargo.toml and moving into the workspace level `Cargo.toml`